sql/executor.go

```diff
@@ -429,6 +439,56 @@ func (p parameters) Arg(name string) (parser.Datum, bool) {
 	}
 }
 
+var _ parser.Args = golangParameters{}
+
+type golangParameters []interface{}
```

Comment:
I modified several functions which were using .exec() and .query(), that demonstrates the usage of the interface. Previously, these invocations used Sprintf to place parameter values directly into the string.

I actually accepted Peter's suggestion, which passes the interfaces directly to `exec()` as a `...interface{}`. I think this is a very easy to use and understand interface for an un-exported method.